cypress/e2e/smoke/create_tx.cy.js

```diff
@@ -44,14 +42,10 @@ describe('[SMOKE] Create transactions tests', () => {
   })
 
   it('[SMOKE] Verify nonce tooltip warning messages', () => {
-    createtx.changeNonce(-1)
-    createtx.verifyTooltipMessage(constants.nonceTooltipMsg.lowerThanCurrent + currentNonce.toString())
```

Comment:
the default nonce is 0, but since we can't enter -1 in a the number field this test can no longer be performed. 
is there a way for the system to think that the current valid nonce is anything other than 0?

Reply:
The test to check the tooltip in case of entered nonce<current nonce was lost. We need to add it again. but with positive nonce ( test data should be also updated in the test safe)